Lab/demo/Lab/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(Lab_Demo)
 include(FeatureSummary)
 
 
-add_definitions ( -DCGAL_NO_DEPRECATED_CODE )
+add_compile_definitions ( -DCGAL_NO_DEPRECATED_CODE )
```

Comment:
Looks like magritte does not like it: https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-6.1-Ic-354/Lab_Demo/TestReport_magritte_arm64-apple_bigsur_clang1200-release-64bits.gz

Reply:
*New in version 3.26: Any leading -D on an item will be removed.* so we have to remove the -D